src/app/proposals/draft/components/stages/SubmitForm.tsx

```diff
@@ -0,0 +1,82 @@
+import DraftPreview from "../DraftPreview";
+import {
+  ProposalDraft,
+  ProposalDraftTransaction,
+  ProposalSocialOption,
+  ProposalChecklist,
+} from "@prisma/client";
+import { useContractRead, useAccount, useBlockNumber } from "wagmi";
+import { ENSGovernorABI } from "@/lib/contracts/abis/ENSGovernor";
+import Tenant from "@/lib/tenant/tenant";
+import RequestSponsorshipForm from "../RequestSponsorshipForm";
+import { useForm, FormProvider } from "react-hook-form";
+import SponsorActions from "../../../sponsor/components/SponsorActions";
+
+// TODO: either read from contract or add to tenant
+const THRESHOLD = 100000000000000000000000;
```

Comment:
This constant is used in several places. Let's ether load it from a contract or move it into constants.